test/hotspot/jtreg/runtime/cds/appcds/sharedStrings/SharedStringsHumongous.java

```diff
@@ -87,7 +87,7 @@ public static void test(String[] args) throws Exception {
         // Extra strings that are humongous are not kelp alive, so they should be GC'ed
         // before dumping the string table. That means the heap should contain no
         // humongous regions.
-        dumpOutput.shouldNotMatch("gc,region,cds. HeapRegion 0x[0-9a-f]* HUM");
+        dumpOutput.shouldNotMatch("gc,region,cds. G1HeapRegion 0x[0-9a-f]* HUM");
```

Comment:
Just a minor nit. I noticed a pre-existing typo on line 87 above. It says "kelp" instead of "kept". Can you fix it?

Reply:
ok